packages/frontend/src/components/Header.jsx

```diff
@@ -13,7 +13,7 @@ export default observer(() => {
     const loadData = async () => {
       await Promise.all([
         unirep.loadAllSignUps(),
-        unirep.loadAllAttestations(),
+        unirep.loadStats(),
         unirep.loadAllEpochs(),
       ])
```

Comment:
I had initially put these function calls here only bc I was trying to check `searchInput` against some data but I didn't have it yet. Then I removed them from `HomePage` bc they were being called twice..
- should these still be called here even though `Header` doesn't need any data?
- if yes, why wouldn't we include `loadAllAttestations` here instead of waiting to call it from `HomePage`?

Reply:
I think `loadStats` can be moved to the home page. I think `loadAllAttestations` should be put wherever attestations are needed (right now the home page). We really need to load attestations using a pagination strategy. Usually I do like, load the first 20, then to request the next 20 pass the last id and ask for posts after that. This is to avoid getting duplicates if new attestations happen between loads.